src/main/java/seedu/stocker/commands/FindCommand.java

```diff
@@ -0,0 +1,79 @@
+package seedu.stocker.commands;
+
+import seedu.stocker.drugs.StockEntry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static seedu.stocker.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+/**
+ * Represents a command to find drugs in the inventory that match a given keyword.
+ */
+public class FindCommand extends Command {
+    public static final String COMMAND_WORD = "find";
+
+    /**
+     * Usage message for the 'find' command.
+     */
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " /n" + ": Finds drug in inventory using name." +
```

Comment:
Unusually long constant. You can consider abstracting out some parts into smaller constants then combining them again,